test/common/README.md

```diff
@@ -116,6 +116,26 @@ Checks whether `IPv6` is supported on this platform.
 
 Checks if there are multiple localhosts available.
 
+### hijackStderr(listener)
+* `listener`
+[&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Normal_objects_and_functions):
+a listener with a single parameter called `data`.
+
+Hijack `process.stderr` to listen `write` action. Once `process.stderr.write` is
+called, `listener` will also be called and the `data` of `write` function will
+be passed to `listener`. What's more, `process.stderr.writeTimes` will plus one
+then.
+
+### hijackStdout(listener)
+* `listener`
+[&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Normal_objects_and_functions):
+a listener with a single parameter called `data`.
+
+Hijack `process.stdout` to listen `write` action. Once `process.stdout.write` is
```

Comment:
Same as above.

IMHO since this is an "internal" guide you could merge then into
```### hijackStdout/Stderr(listener)```